create_pnor_image.pl

```diff
@@ -150,7 +150,7 @@
 print "pnor_data_dir = $pnor_data_dir\n";
 
 my $build_pnor_command = "$hb_image_dir/buildpnor.pl";
-$build_pnor_command .= " --pnorOutBin $pnor_filename --pnorLayout $xml_layout_file";
+$build_pnor_command .= " --pnorOutBin $pnor_filename --pnorLayout $xml_layout_file --editedLayoutLocation $scratch_dir";
```

Comment:
only other thought I had is .. does it make sense to generate the image 2x?  1 time in the update_image would suffice I'd think.

Reply:
Will the file differ between those two times that it is called?
If not, then I guess we pick only one place to store the edited layout file.

Reply:
it should not differ; it should process the same input into the same output.  Otherwise the pnor packaging would be making different size/offset assumptions than the update tool.

Reply:
Ok then It will only save the edited XML in the update_image script.